src/tests/GoodsDetail.test.tsx

```diff
@@ -0,0 +1,60 @@
+import { ChakraProvider } from '@chakra-ui/react';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { BrowserRouter } from 'react-router-dom';
+
+import { GoodsDetail } from '@/components/features/Goods/Detail';
+import { OptionSection } from '@/components/features/Goods/Detail/OptionSection';
+
+const queryClient = new QueryClient();
+
+const productData = {
+  id: 3245119,
+  name: '[단독각인] 피렌체 1221 에디션 오드코롱 50ml (13종 택1)',
+  imageUrl:
+    'https://st.kakaocdn.net/product/gift/product/20240215083306_8e1db057580145829542463a84971ae3.png',
+  price: 145000,
```

Comment:
소소하게 큰 숫자는 numeric separtor를 사용해주시면 보기 좋을거 같네요!
```suggestion
  price: 145_000,
```

Reply:
넵 감사합니다!